src/features/commit/components/CommitValidationMark.jsx

```diff
@@ -0,0 +1,15 @@
+import PropTypes from "prop-types";
+
+const CommitQualityScore = ({ qualityScore }) => {
+  return (
+    <div className={`w-20 rounded px-1 py-1 text-xl font-bold text-blue-500`}>
```

Comment:
```suggestion
    <div className="w-20 rounded px-1 py-1 text-xl font-bold text-blue-500">
```
지정된 스타일만 있으니 이렇게 변경하는 것은 어떨까요~!

Reply:
네 ! 그러네요, 
이전에 style 을 동적으로 하기 위해서 `` 백틱을 썼었는데, 해당부분 지우게 되면서 저 부분을 남겨두었네요.
수정하겠습니다!